app/helpers/admin/policy_configurations_helper.rb

```diff
@@ -1,8 +1,7 @@
 module Admin
   module PolicyConfigurationsHelper
     def options_for_academic_year
-      years = 4
-      (0..(years - 1)).map { |relative_year| AcademicYear.current + relative_year }
+      (0..3).map { |relative_year| AcademicYear.current + relative_year }
```

Comment:
@nomasprime So when the current `AcademicYear` is `2022`, from this method we will still get 4 new `AcademicYears`?

```
(0..3).map { |relative_year| AcademicYear.current + relative_year }
=> [#<AcademicYear:0x000055c3ddba6430 @start_year=2022, @end_year=2023>, #<AcademicYear:0x000055c3ddba6070 @start_year=2023, @end_year=2024>, #<AcademicYear:0x000055c3ddba5d28 @start_year=2024, @end_year=2025>, #<AcademicYear:0x000055c3ddba59e0 @start_year=2025, @end_year=2026>]
```

I am not sure this is what we are trying to achieve. For ECP it runs until the final `AcademicYear` of 2024/2025.

So the number in the array returned should be decreasing by 1 each year, should it not?

Reply:
Technically this should be tied to those years but business was fine with the next four years. Should raise a ticket though.